internal/c/parts/audio/extras/build.mk

```diff
@@ -34,14 +34,22 @@ LIBXMP_OBJS += $(patsubst %.c,$(PATH_INTERNAL_C)/parts/audio/extras/libxmp-lite/
 LIBXMP_LIB := $(PATH_INTERNAL_C)/parts/audio/extras/libxmp-lite.a
 
 $(PATH_INTERNAL_C)/parts/audio/extras/libxmp-lite/%.o: $(PATH_INTERNAL_C)/parts/audio/extras/libxmp-lite/%.c
-	$(CC) $(CFLAGS) -Wall -DLIBXMP_CORE_PLAYER -DLIBXMP_NO_PROWIZARD -DLIBXMP_NO_DEPACKERS -DBUILDING_STATIC $< -c -o $@
+	$(CC) -O2 $(CFLAGS) -Wall -DLIBXMP_CORE_PLAYER -DLIBXMP_NO_PROWIZARD -DLIBXMP_NO_DEPACKERS -DBUILDING_STATIC $< -c -o $@
 
 $(LIBXMP_LIB): $(LIBXMP_OBJS)
 	$(AR) rcs $@ $^
 
+HIVELY_SRCS := hvl_replay.c
```

Comment:
It would probably be worth making a new issue for the addition of AHX and HVL support. You can keep the code in this PR, but having the issue will make it easier for people searching for this change.

Reply:
Done.